zap/src/test/resources/org/zaproxy/zap/spider/parser/html/ImgElementsSpiderHtmlParser.html

```diff
@@ -22,6 +22,8 @@
 <img src="mailto:img@example.com">
 <img src="javascript:hello();">
 <img src="scheme://img.example.com/invalid">
+<img lowsrc="/img/lowresolution">
+<img dynsrc="https://video.example.com/native/video">
```

Comment:
You seem to have added these to the set of URLs which are expected to be ignored.

Reply:
Thanks, overlooked :)

Reply:
No problem, been there 😃